sys/net/link_layer/ieee802154/submac.c

```diff
@@ -40,25 +61,9 @@ static inline void _req_set_trx_state_wait_busy(ieee802154_dev_t *dev,
      */
     do {
         res = ieee802154_radio_request_set_trx_state(dev, state);
-    }
-    while (res == -EBUSY);
-    assert(res >= 0);
-}
-
-static void _tx_end(ieee802154_submac_t *submac, int status,
-                    ieee802154_tx_info_t *info)
-{
-    ieee802154_dev_t *dev = &submac->dev;
-
-    ieee802154_trx_state_t next_state = submac->state == IEEE802154_STATE_LISTEN
-                                        ? IEEE802154_TRX_STATE_RX_ON
-                                        : IEEE802154_TRX_STATE_TRX_OFF;
-
-    _req_set_trx_state_wait_busy(dev, next_state);
-    submac->wait_for_ack = false;
-    submac->tx = false;
+    }while (res == -EBUSY);
```

Comment:
the problem is that `uncrustify` is buggy. I would avoid using it without manual verification.